src/sharedServices/queue/sendSqsMessage.ts

```diff
@@ -7,20 +7,29 @@ import { getEnv } from '../../utils/helpers'
 
 export const sendSqsMessage = async (
   messageBody: object,
-  queueUrl: string
+  queueUrl: string,
+  delaySendInSeconds: number | undefined = undefined
 ): Promise<string | undefined> => {
-  return sendSqsMessageWithStringBody(JSON.stringify(messageBody), queueUrl)
+  return sendSqsMessageWithStringBody(
+    JSON.stringify(messageBody),
+    queueUrl,
+    delaySendInSeconds
+  )
 }
 
 export const sendSqsMessageWithStringBody = async (
   messageBody: string,
-  queueUrl: string
+  queueUrl: string,
+  delaySendInSeconds: number | undefined = undefined
```

Comment:
could we just do `delaySendInSeconds?: number`

Reply:
Yes, good point